src/FHIRConverterAPI/Models/FHIRConverterRequest.cs

```diff
@@ -0,0 +1,14 @@
+public class FHIRConverterRequest
+{
+    // The message to be converted as a string.
+    public string input_data { get; set; }
+
```

Comment:
there are a lot of lint warning on these files - should they be listened to or ignored?